Britannia/localization/english/event_localization/BTWK_events_l_english.yml

```diff
@@ -25,26 +25,40 @@
-  BTWK_frontier_event.0202_desc_imperial_like_destroyer:0 "[destroyed_hadrians_wall_char.GetFirstName] has done this despite its immense historical and cultural importance. Many are furious at what they see as a callous disregard - by one of their own kin no less - for the island's Roman heritage."
+  BTWK_frontier_event.0202_desc_imperial_like_destroyer:0 "[destroyed_hadrians_wall_char.GetFirstName] has done this despite its immense historical and cultural importance. Many are furious at what they see as a callous disregard - by one of their own no less - for the island's Roman heritage."
   BTWK_frontier_event.0202_desc_else_destroyer:0 "While [destroyed_hadrians_wall_char.GetFirstNamePossessiveNoTooltipRegnal] seems to have regarded this as purely a practical move, Britons and Latins alike are furious at what they see as a callous disregard for the island's Roman heritage."
   BTWK_frontier_event.0202.a_imperial_like:0 "A terrible day for Britannia"
   BTWK_frontier_event.0202.a_other_0 "Interesting news" 
   BTWK_frontier_event.0202.a_other_vassal:0 "For the good of the realm!"
 
   BTWK_frontier_event.0203:0 "Hadrian's Wall Restored"  # ROOT of restoration
-  BTWK_frontier_event.0203_desc:0 "The conclusion of the renovations in [GetGlobalVar('last_old_hadrians_wall_province').MakeScope.Province.GetProvince] has marked the complete restoration of Hadrian's Wall. While the gesture was ultimately symbolic as the wall no longer marks the northern frontier of Britannia, the wall is now operable as the defensive fortification it once was, should the need ever arise again. \n\nThough the effort to rebuild has taken many years, gold and manpower to complete, as the overseer of the final stones being set into place, you are the one the poems shall remember the most. Many among the isle are beginning to praise you as 'Hadrianus', as a successor to the great Emperor Hadrian himself. Perhaps this new reputation might be the making of a new High King..."
-  BTWK_frontier_event.0203.a:0 "OPTION"  # Not sure on this
+  BTWK_frontier_event.0203_desc:0 "The conclusion of the renovations in [GetGlobalVar('last_old_hadrians_wall_province').MakeScope.Province.GetProvince] has marked the complete restoration of Hadrian's Wall. While the gesture was ultimately symbolic (as the wall no longer marks the northern frontier of Britannia), it is still practically useful as a defensive fortification, should the need ever arise. \n\nThough the effort to rebuild has taken many years, gold and manpower to complete, as the overseer of the final stones being set into place, you are the one the poems shall remember the most. Many among the isle are beginning to praise you as 'Hadrianus', as a successor to the great Emperor Hadrian himself." 
```

Comment:
Is the localisation issue with [GetGlobalVar('last_old_hadrians_wall_province').MakeScope.Province.GetProvince]?

Reply:
Yes. Obviously I've been away for a month so I need to rediagnose the problem appropriately, but the short of it was it just didn't work. I'll have a look now properly and let you know what the issue is, if it's still there (who knows, it might have magically fixed itself!)